### PR TITLE
Aeyb #42 calendar attendance monitoring

### PR DESCRIPTION
**Pull request by long715**

### CODE REVIEW

File: frontend/src/pages/calendar/components/WeeklyInstance.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import MeetingDTO from '../../../shared/Types/dtos/MeetingDTO';
+
+function WeeklyInstance(props: { meeting: MeetingDTO }) {
```
Comment (grantliunz) [inline frontend/src/pages/calendar/components/WeeklyInstance.tsx:3]:
```javascript
    function nth(n) { return ["st", "nd", "rd"][((n + 90) % 100 - 10) % 10 - 1] || "th" }
    const days = ["Sunday ", "Monday ", "Tuesday ", "Wednesday ", "Thursday ", "Friday ", "Saturday "];
    const date = new Date(props.meeting.time);
```

File: frontend/src/pages/calendar/components/WeeklyInstance.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <div>
+            <div className='flex justify-center overflow-scroll'>
+                <div className={`flex-col mb-4 w-full lg:w-3/4 py-4 px-4  rounded-lg text-[#262B6C] bg-slate-100`}>
+                    <p className='font-bold text-3xl'>{props.meeting.name}</p>
+                    <div className='text-xl text-[#464a83]'>{
+                        props.meeting.description === undefined ?
+                            <p>no description.</p>
+                            : <p>{props.meeting.description}</p>
+                    }</div>
+                    <p className='text-xl mt-4'>{props.meeting.location}</p>
```
Comment (grantliunz) [inline frontend/src/pages/calendar/components/WeeklyInstance.tsx]:
```javascript
{days[date.getDay()]} {date.getDate()}{nth(date.getDate())} at {date.getHours() % 12}:{date.getMinutes() < 10 ? "0" + date.getMinutes() : date.getMinutes()} {date.getHours() >= 12 ? "PM" : "AM"}
```

**Review by grantliunz — CHANGES_REQUESTED** [review]
Overall look great!
Just thought it would be better to also show the date and time of the meeting on each meeting section as well. Have suggested some code that will do that :)

File: frontend/src/pages/calendar/components/WeeklyInstance.tsx (hunk 2 above)
Comment (long715) [reply frontend/src/pages/calendar/components/WeeklyInstance.tsx]: thank you! i've added the change

**Review by grantliunz — APPROVED** [review]
lgtm!

**Pull request merged**